jupyterhub/schema.yaml

```diff
@@ -429,7 +429,7 @@ properties:
           some custom way.
 
           For more details, see the [Kubernetes
-          documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes).
```

Comment:
The `#notes` anchor was removed from the hyperlink. This was leading to a link check failure in CI, so we removed the anchor.